homeassistant/components/rituals_perfume_genie/translations/nl.json

```diff
@@ -0,0 +1,21 @@
+{
+    "config": {
```

Comment:
Please do not add additional languages. These additional translations can be managed via Lokalised after this has been merged.

Reply:
Removed the additional translation in commit https://github.com/home-assistant/core/pull/46218/commits/67bad89e7e284e895a958be9d8fb7c9f4d5abed1.